controller_manager/controller_manager/spawner.py

```diff
@@ -266,6 +263,38 @@ def main(args=None):
                         )
                         return 1
 
+                if param_file:
+                    parameter = Parameter()
+                    parameter.name = prefixed_controller_name + ".params_file"
+                    parameter.value = get_parameter_value(string_value=param_file)
+
+                    response = call_set_parameters(
```

Comment:
So, we don't need to load parameter file on our own anymore, but we can just give a file as a parameter to this method. Correct?
This is cool! :sunglasses:

Reply:
Yup. Exactly